api/source/job_control/aggregate_statistics.rst

```diff
@@ -148,36 +154,31 @@ grouped by week or month and filtered by a time range.
     {
         "data": [
             {
-                "billable_minutes": 10.0,
-                "start_date": "2015-03-26T11:36:09.237373",
-                "end_date": "2015-03-31T23:59:59.999999"
+                "billable_minutes_total": 15,
```

Comment:
Could you make the example response have more than just the billable_minutes_total metric so they can see how it'll look with multiple metrics?